.gitignore

```diff
@@ -13,6 +13,8 @@ classesEclipse
 deps/
 eclipse/
 bin/
+.vscode/
+
```

Comment:
Please do not add unrelated changes, modify the gitignore with IDE-specific settings.